python/ngraph/ops.py

```diff
@@ -60,6 +60,23 @@ def constant(value, dtype=None, name=None):  # type: (NumericData, NumericType,
     return make_constant_node(value, dtype)
 
 
+@nameable_op
+def elu(data, alpha, name=None):  # type: (NodeInput, NodeInput, str) -> Node
+    """Perform Exponential Linear Unit operation element-wise on data from input node.
+
+    Computes exponential linear: alpha * (exp(data) - 1) if < 0, data otherwise.
+
+    For more information refer to:
+    `Fast and Accurate Deep Network Learning by Exponential Linear Units (ELUs) <http://arxiv.org/abs/1511.07289>`_
```

Comment:
Some artifact `_` at the end of line?

Reply:
No, that's how ReStructuredText represents links.